src/transformers/tokenization_utils_fast.py

```diff
@@ -863,3 +864,99 @@ def train_new_from_iterator(
             kwargs["additional_special_tokens"] = additional_special_tokens
 
         return self.__class__(tokenizer_object=tokenizer, **kwargs)
+
+    def _update_normalizer(self):
+        """Updates the underlying normalizer with the current `add_prefix_space` and `legacy` settings."""
+        sequence = json.loads(normalizers.Sequence([]).__getstate__())
+        final_sequence = normalizers.Sequence([])
+        if self._tokenizer.normalizer is not None and type(self._tokenizer.normalizer) in (
+            normalizers.Prepend,
+            normalizers.Precompiled,
+        ):
+            # If normalizer is not a Sequence, add it to a sequence
+            sequence["normalizers"].append(json.loads(self._tokenizer.normalizer.__getstate__().decode("utf-8")))
```

Comment:
if we have a prepend normalizer, we want to remove it (of course everything should be in the `if legacy = False` ). 

Here again, we only want to update for people who do not want to convert from slow / load the fast tokenizer but legacy is false